helm-chart/templates/hive.yaml

```diff
@@ -0,0 +1,74 @@
+# templates/hive.yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-hive
+  namespace: {{ .Values.global.namespace }}
+  labels:
+    app: {{ .Release.Name }}-hive
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-hive
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-hive
+    spec:
+      containers:
+      - name: hive
+        image: "{{ .Values.hive.image.repository }}:{{ .Values.hive.image.tag }}"
+        ports:
+        - containerPort: 3306
+        - containerPort: 9000
+        - containerPort: 9083
+        - containerPort: 50070
+        readinessProbe:
+          exec:
+            command: ["/tmp/check-status.sh"]
+          periodSeconds: 10
+          timeoutSeconds: 60
+          failureThreshold: 5
+        env:
+        - name: HADOOP_USER_NAME
+          value: root
```

Comment:
The environment variable `HADOOP_USER_NAME` is hard code in yaml template. Since  `HADOOP_USER_NAME` is also defined in spark , trino  yaml template  and these values is required to be the same in playground environment. I suggest replace the value with placeholder in template yaml and set the default value "root" in values.yaml.

Reply:
fixed.